Test_Tshirt.cpp

```diff
@@ -0,0 +1,14 @@
+#include <cassert>
+#include <iostream>
+#include "TshirtSizeClassifier.h"
+#include "Test_Tshirt.h"
+
+void TestForTshirtSizeClassifier() {
+    assert(SizeChecker(37) == 'S');
+    assert(SizeChecker(38) == 'S');

```

Comment:
Why is 38 expected to be `S`? In what way will this affect the user's experience?